tools/tu_collector/tu_collector/tu_collector.py

```diff
@@ -367,13 +377,20 @@ def main():
                              "actions of which the compiled source file "
                              "matches this path. E.g.: /path/to/*/files")
 
+    parser.add_argument('-v', '--verbose',
+                        action='store_true',
+                        dest='verbose',
+                        help="Set verbosity level.")
```

Comment:
I think a little change to the help message would be nice you can not set it just enable debug level logging with this flag.